src/java.base/share/classes/java/lang/StringUTF16.java

```diff
@@ -1532,18 +1528,17 @@ public static boolean contentEquals(byte[] value, CharSequence cs, int len) {
         return true;
     }
 
-    public static int putCharsAt(byte[] value, int i, char c1, char c2, char c3, char c4) {
+    public static void putCharsAt(byte[] value, int i, char c1, char c2, char c3, char c4) {
         int end = i + 4;
         checkBoundsBeginEnd(i, end, value);
         putChar(value, i++, c1);
         putChar(value, i++, c2);
         putChar(value, i++, c3);
         putChar(value, i++, c4);
         assert(i == end);
```

Comment:
```suggestion
        checkBoundsBeginEnd(i, end, value);
        putChar(value, i    , c1);
        putChar(value, i + 1, c2);
        putChar(value, i + 2, c3);
        putChar(value, i + 3, c4);
```

StringUTF16 can also use `+1 +2 +3` to replace `++`